docusaurus/docs/publish-a-plugin/provide-provisioning.md

```diff
@@ -0,0 +1,60 @@
+---
+id: provide-provisioning
+title: Provision your plugin
```

Comment:
I'm not sure about the focus being on "provision(ing)" - the mechanism vs "Provide a test environment" - the intent.

Reply:
If Marcus is okay with it, I'm good with "Provide a test environment".